test/test_python_dispatch.py

```diff
@@ -562,6 +562,30 @@ def test_autograd_in_attr(self):
         self.assertIsNone(t.grad)
         self.assertIsNotNone(t.elem.grad)
 
+    def test_multiple_ops_subclass(self):
+        # This is a Direct Subclass, don't do that!
```

Comment:
This is a direct subclass in the sense that the Tensor that we send to the backend and the subclass are the same.
It is ok to do it, but not composable and tricky in general. Since a lot of people use these tests as example, I prefered to warn here that this is not a good example to copy.

Reply:
^ you should probably put that into the comment in a code, as a user I would be more even confused haha

Reply:
related question: does the bug with the non-stack-based version only show up when you do a "Direct Subclass" (and not show up when you do a wrapper subclass)?

Reply:
The direct subclass is the simplest repro I could come up with.
@Chillee managed to hit the same problem with some of the functorch tests. In that case, it was while going down a `clamp_min_` op, an extra `clone` was done before executing the function itself. So I guess functionalization was involved? But I don't know how to enable that easily in a small repro. Hence the use of direct subclass and conjugate fallback.